app/jobs/course/material/folder/downloading_job.rb

```diff
@@ -0,0 +1,12 @@
+class Course::Material::Folder::DownloadingJob < ApplicationJob
```

Comment:
Somehow that feels strange. `Course::Material::FolderDownloadJob`? maybe I shouldn't have used the participle in my grader?